pkg/reconciler/v1alpha1/clusteringress/clusteringress_test.go

```diff
@@ -489,18 +489,15 @@ func newTestSetup(t *testing.T, configs ...*corev1.ConfigMap) (
 	configMapWatcher *configmap.ManualWatcher) {
 
 	kubeClient = fakekubeclientset.NewSimpleClientset()
-	cms := []*corev1.ConfigMap{
+	cms := append([]*corev1.ConfigMap{
```

Comment:
Use {{

Reply:
Done.
I wasn't thrilled with NullTracker, since it's kind of useless, but did anyway :)